### PR TITLE
docs: minor upgrades

### PR DESCRIPTION
**Pull request by danpastori**
### 🔗 Linked issue

#3769 

### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description
This file is a suggestion for all of the small upgrades that take place from Nuxt 2 to Nuxt 3. Features that are now composables instead of global variables, component names, etc. Features that don't warrant an entire page. If you like this approach, I can add a lot of these smaller upgrades as I get to them. Just let me know and no rush!

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/content/1.getting-started/6.migration/9.minor-upgrades.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Minor Upgrades
+There are few minor upgrades that don't require an entire page dedicated to the process, but are still very important.
+
+## Upgraded NuxtLink component
+Most of the syntax and functionality are the same for the global [NuxtLink](https://v3.nuxtjs.org/docs/usage/nuxt-link#nuxtlink) component. However, the component name has changed to have capital letters instead of hyphenated. Youu will have to update this component throughout your app.
```
Comment (danielroe) [inline docs/content/1.getting-started/6.migration/9.minor-upgrades.md:5]: This shouldn't be needed - it should work either way.

Comment (pi0) [reply docs/content/1.getting-started/6.migration/9.minor-upgrades.md:5]: (Small note: NLink is unsupported but i guess will add it back either deprecated or support again.)

**Review by danielroe — APPROVED** [review]
Thanks! ❤️

**Comment by Atinux** [comment]
Thanks @danpastori :heart:

**Pull request merged**